README.md

```diff
@@ -36,11 +36,9 @@ git checkout master
 
 2. Set up the Engine development environment: See [the wiki page](https://github.com/flutter/flutter/wiki/Setting-up-the-Engine-development-environment).
 
-3. In **the engine repo**, add the prototype remote and switch to the prototype branch:
+3. In **the engine repo**, apply this playground's patches:
 ```
-git remote add dkwingsmt https://github.com/dkwingsmt/engine/
-git fetch dkwingsmt fffa6e19f5b71bedd55ac3a7fa6acfd85eda5983
-git checkout fffa6e19f5b71bedd55ac3a7fa6acfd85eda5983
+git apply /path/to/playground/patches/001-Add-multi-view-Flutter-Windows-C++-APIs.patch
```

Comment:
Should the instructions here say something like: "For Windows use this patch:" - since macOS will likely be a different patch file?

Reply:
Updated! We can reconsider this if/when another platform also needs patch files.

Reply:
This feels like you have to clone the engine repo to make the app work. In fact, to make the app download the files from the custom engine, I suppose the engine has to be fully compiled, which is not very available to many developers.

On the other hand, the developer can also simply override the files in the `ephemeral` directory, right? Although this means that they'll have to repatch it every time the engine is updated. Should we mention this too? Or is there a better way?

Reply:
Are you asking because you'd like Flutter macOS to use the "vanilla" engine without needing to a local compilation? Perhaps it would be better to have separate instructions for platforms that can use the "vanilla" engine and platforms that need a "custom" engine with patches.

To answer your question directly, yup the `ephemeral` directory approach could work as well. However, the instructions on how to maintain that are a little grosser. Also, the ergonomics there are a bit tricky - you need to build once using the `flutter` tool and then drop down to native tooling for subsequent builds.

Personally, I think the patch file approach is the best for Windows. It is a workflow that a Flutter engine developer is familiar with. Flutter Windows multi-view isn't productionized yet, you currently still need Flutter engine expertise to use it.